.github/workflows/build-macos.yml

```diff
@@ -16,6 +16,10 @@ jobs:
     timeout-minutes: 60
     env:
       RUNNER_TYPE: ${{ matrix.runner[0] }}
+      # Disable ccache as it seemed to have weird behavior and caused significant slowdown.
+      # Reenable if needed
+      # TRITON_BUILD_WITH_CCACHE: "true"
+      TRITON_BUILD_WITH_CLANG_LLD: "TRUE"
```

Comment:
Have you tried a run with ccache and lld enabled?

Reply:
That's what I proposed

Reply:
Should be clean now